esp/src/src/Utility.ts

```diff
@@ -3,53 +3,62 @@ import { format as d3Format, Palette } from "@hpcc-js/common";
 import { Level, join } from "@hpcc-js/util";
 import * as arrayUtil from "dojo/_base/array";
 import * as domConstruct from "dojo/dom-construct";
-import * as entities from "dojox/html/entities";
 import { darkTheme } from "../src-react/themes";
 import nlsHPCC from "./nlsHPCC";
 
 declare const dojoConfig;
 declare const ActiveXObject;
 declare const require;
 
-export function xmlEncode(str) {
```

Comment:
There are places where these are used in the v5 UI. GraphWidget.js uses xmlEncode, and InfoGridWidget.js uses xmlEncode2